server/src/main/java/org/opensearch/common/settings/GenericSettingUpgrader.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
```

Comment:
@saratvemulapalli Wanted to double check that I don't need the "Licensed to Elasticsearch" section in this new file. Correct?

Reply:
Correct, the only license header we would need is SPDX for all new files.

Reply:
And the "Modifications Copyright OpenSearch Contributors." part?

Reply:
Yup that is enforced by the license header check and its needed.